rpc/answerer_stats.go

```diff
@@ -0,0 +1,88 @@
+package rpc
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/edaniels/golog"
+	"github.com/pion/webrtc/v3"
+)
+
+// timeFormatStr copied from DefaultTimeFormatStr in RDK.
+const timeFormatStr = "2006-01-02T15:04:05.000Z0700"
+
+// answererStats is a collection of measurements/information gathered during
+// the course of a single connection establishment attempt from a signaling
+// answerer. We log the contents of the struct as a single INFO log message
+// upon successful or failed connection establishment to avoid emitting dozens
+// of logs during the answering process and cluttering regular robot logs.
+// Answerer stats are only logged in production for external signalers.
+type answererStats struct {
+	// mu guards all fields on answererStats.
+	mu sync.Mutex
+
+	success bool
+
+	// Stats below will be logged.
+	answerRequestInitReceived *time.Time
+	numAnswerUpdates          int
+	localICECandidates        []*localICECandidate
```

Comment:
local refers to client side and remote would be server side? or is it the other way around? might be good to clarify here

Reply:
Ah yeah confusing. Local is the signaling answerer's "local", so technically server-side. Remote is what's received from the client, so client-side. Will add a comment to clarify.